app/components/treatment/voting/modal.js

```diff
@@ -33,9 +33,33 @@ export default class TreatmentVotingModalComponent extends Component {
   @restartableTask
   /** @type {import("ember-concurrency").Task} */
   *fetchStemmingen() {
-    this.stemmingen = (yield this.args.behandeling.stemmingen).sortBy(
-      'position'
-    );
+    const stemmingen = [];
+    const pageSize = 20;
+
+    const firstPage = yield this.store.query('stemming', {
+      'filter[behandeling-van-agendapunt][:id:]': this.args.behandeling.id,
+      sort: 'position',
+      page: {
+        size: pageSize,
```

Comment:
Wouldn't be easier to fetch with an absurd pageSize? like 999

Reply:
Yeah I'm still hesitating what the best approach is.